src/app/components/MarketDataHeader.tsx

```diff
@@ -0,0 +1,103 @@
+import { CoinsIcon } from "../icons/CoinsIcon";
+import { ExchangeIcon } from "../icons/ExchangeIcon";
+import { ChevronUpIcon } from "../icons/ChevronUpIcon";
+import { BitcoinIcon } from "../icons/BitcoinIcon";
+import { EthereumIcon } from "../icons/EthereumIcon";
+import { ChevronDownIcon } from "../icons/ChevronDownIcon";
+import formatNumber from "@/app/utils/formatNumber";
+import getPercentage from "@/app/utils/getPercentage";
+import MarketPercentageBar from "./MarketPercentageBar";
+
+async function getMarketData() {
+  const res = await fetch(
+    "https://api.coingecko.com/api/v3/global?x_cg_demo_api_key=CG-q2aLp44XjWCoV41TwyFeLSnE",
+    {
+      next: {
+        revalidate: 0,
+      },
+    }
+  );
+  return res.json();
+}
+
+export default async function MarketDataHeader() {
+  const marketData = await getMarketData();
+  const {
+    active_cryptocurrencies,
+    markets,
+    total_market_cap,
+    total_volume,
+    market_cap_percentage,
+    market_cap_change_percentage_24h_usd,
+  } = marketData.data;
+
+  const percentageVolumeBasedOnTotalMarketCap = getPercentage(
+    total_volume.btc,
+    total_market_cap.btc
+  );
+  const btcMarketCapPercentage: number = Math.floor(market_cap_percentage.btc);
+  const ethMarketCapPercentage: number = Math.floor(market_cap_percentage.eth);
```

Comment:
```suggestion
  const btcMarketCapPercentage = Math.floor(market_cap_percentage.btc);
  const ethMarketCapPercentage = Math.floor(market_cap_percentage.eth);
```

Explicitly mentioning the return type is redundant here because Math.floor will always return a number type.